models/models.js

```diff
@@ -9,14 +9,36 @@ exports.selectCategories = () => {
       return categories;
     })
     .catch((err) => {
-      return err;
+      throw err;
```

Comment:
This catch block isn't necessary in the model. If this promise fails, there will be a knock on effect which means our catch block in the controller will handle it.